core/domain/question_domain.py

```diff
@@ -229,6 +222,20 @@ def validate(self):
             )
         self.question_state_data.validate({}, False)
 
+    def validate(self):
+        """Validates the Question domain object before it is saved."""
+
+        if not isinstance(self.id, basestring):
+            raise utils.ValidationError(
+                'Expected ID to be a string, received %s' % self.id)
+
+        if not isinstance(self.version, int):
```

Comment:
Check >= 1

Reply:
Done

Reply:
Looks like this change broke a lot of the existing checks (apparently version was 0 in the question tests... should I make the check for greater than 0?)